home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/exception/PasswordException.java

```diff
@@ -0,0 +1,7 @@
+package com.softserveinc.ita.homeproject.homeservice.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.NOT_ACCEPTABLE, reason = "Password too weak")
+public class PasswordException extends RuntimeException { }
```

Comment:
Every custom exception should be extended at least by BaseHomeException

Reply:
Updated to extend `BaseHomeException`.